lib/screens/aim/aim.dart

```diff
@@ -0,0 +1,138 @@
+import 'package:ecellapp/core/res/colors.dart';
+import 'package:ecellapp/screens/aim/text_aim.dart';
+import 'package:ecellapp/widgets/screen_background.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:ecellapp/core/res/dimens.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class AimScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(
+        children: [
+          ScreenBackground(
+            elementId: 0,
+          ),
```

Comment:
Reduce number of lines wherever possible
```suggestion
          ScreenBackground(elementId: 0),
```